config/config.yaml

```diff
@@ -12,10 +12,11 @@ kraken:
             - scenarios/arcaflow/cpu-hog/input.yaml
             - scenarios/arcaflow/memory-hog/input.yaml
             - scenarios/arcaflow/io-hog/input.yaml
+            - scenarios/arcaflow/io-hog/input.yaml
         - application_outages:
             - scenarios/openshift/app_outage.yaml
         - container_scenarios:                             # List of chaos pod scenarios to load
-            - - scenarios/openshift/container_etcd.yml
+            - scenarios/openshift/container_etcd.yml
         - plugin_scenarios:
```

Comment:
The suggested change is out of scope of this PR but would be good to add as part of the refactoring: 

It would be easy for the users to understand if we rename plugin_scenarios to a appropriate name describing the scenario instead - pod_disruptions, hog_scenarios for instance. Also might be better to have the configs under scenarios/openshift and scenarios/kubernetes instead of scenarios/arcaflow to have everything under similar dir structure and avoid any confusion for the end users.

Thoughts?